src/widgets/clear-all/clear-all.js

```diff
@@ -51,6 +51,10 @@ const renderer = ({
   );
 };
 
+const disposer = containerNode => () => {
```

Comment:
Why not inline this function or make the implementation part of the utils function?